sdk/security/keyvault/internal/challenge_policy_test.go

```diff
@@ -22,6 +22,22 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	challengedToken = "needs more claims"
+	claimsToken     = "all the claims"
+	kvChallenge     = `Bearer authorization="https://login.microsoftonline.com/tenant", resource="https://vault.azure.net"`
+	caeChallenge1   = `Bearer realm="", authorization_uri="https://login.microsoftonline.com/common/oauth2/authorize", error="insufficient_claims", claims="dGVzdGluZzE="`
```

Comment:
Sorry to nitpick but most of these should be declared `const`. Also, a package-level block should contain only symbols used by multiple tests. `requireToken` for example is better declared in `TestChallengePolicy_CAE` because it's only used there

Reply:
fixed in 8de3f62